src/containers/Review/assignment/AssignmentSectionRow.tsx

```diff
@@ -154,12 +152,11 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = ({
       {isReassignment && (
         <Grid.Row>
           <Reassignment
-            assignments={assignments}
             sectionCode={sectionCode}
-            isLastLevel={isLastLevel}
-            previousAssignee={assignmentOptions.selected}
+            isLastLevel={isLastLevel()}
             assignedSections={assignedSections}
             setAssignedSections={setAssignedSections}
+            assignmentOptions={assignmentOptions}
```

Comment:
There was some extra variables not needed for Re-assignment

I was investigating why it used to show only the same assignment on the dropdown, I found out the problem and ended up removing some extra props that could be already used with `assignmentOptions`